internal/MozziGuts_impl_RP2040.hpp

```diff
@@ -288,6 +260,7 @@ static void startAudio() {
   micros_per_update = 1000000l / MOZZI_AUDIO_RATE;
   do {
     next_audio_update = make_timeout_time_us(micros_per_update);
+    next_audio_update_shifted = to_us_since_boot(next_audio_update);
```

Comment:
I think this is not correct (but wasn't before, either): ```startMozzi()``` may not necessarily be called a time 0. Rather (I think):

```
next_audio_update_shifted = make_timeout_time_us(delayed_by_us(time_us_64(), micros_per_update) << 8;
```

Reply:
Are you sure? I am okay to admit that I forgot a shift there, so more like:

```
next_audio_update_shifted = to_us_since_boot(next_audio_update) <<8;
```

But, the `next_audio_update = make_timeout_time_us(micros_per_update);` increment from the time it is called on (does not need to be zeros), but returns an *absolute time*. `delayed_by_us` will return an absolute time, in the future already, and the `make_timeout_time_us` will give a "timestamp a number of microseconds from the current time.", hence a absolute time, starting at the current time and delayed by (current_time + micros_per_update), returned by the `delayed_by_us`. I might be wrong though because I am not sure I understand all the `absolute_time_t` correctly. The shift is missing though.

Reply:
You are right, I got confused with relative and absolute timestamps!